server/commands/config/config.go

```diff
@@ -0,0 +1,60 @@
+package config
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/nathanaelhoun/mattermost-plugin-circleci/server/store"
+)
+
+const (
+	// Command name of the command
+	Command = "config"
```

Comment:
Rename to CommandTrigger to distinguish with the model.Command struct
```suggestion
	CommandTrigger = "config"
```